addon/scheduler.ts

```diff
@@ -27,6 +28,12 @@ const waiter = buildWaiter('ember-app-scheduler-waiter');
 
 reset();
 
+class Scheduler {
+  @tracked isIdle = false;
+}
+const scheduler = new Scheduler();
+export default scheduler;
```

Comment:
Only reason I'm creating a class here is to setup auto-tracking on `isIdle`. Not sure if there is a more elegant way to do this in a functional module.

Reply:
Nope, this is perfectly fine!

Reply:
I think having this class is totally fine. I wouldn't put it here though. I'd put it in the helper file itself though, not here. It feels odd here.

Reply:
Actually ignore this. We need this here since we need to set `scheduler.isIdle`.